marda_registry/data/extractors/quantumespresso.yml

```
@@ -0,0 +1,40 @@
+---
+id: qe-tools
+name: Tools for Quantum ESPRESSO
+description: A set of tools for the Quantum ESPRESSO suite of electronic-structure simulation codes.
+subject:
+    - atomistic simulation
+    - electronic structure calculations
+    - materials modeling
+source_repository: https://github.com/aiidateam/qe-tools
+usage:
+    - method: python
+      setup: qe_tools
+      command: qe_tools.extractors.extract({{ input_path }}, {{ input_type }})
+installation:
+    - method: pip
+      packages:
+          - qe-tools
```

maybe add a lower pin here for when marda support was added?

```suggestion
          - "qe-tools > 2.1" 
```

---

Reverting to git reference (`aiidateam`'s) until official release